### DISCUSSION TITLE
There are some problems with the routing templates, and there are some problems with the document

### DISCUSSION THREAD

**Discussion by azimxxm** [flet-dev/flet#1013] [Q&A]
Route templates when getting to know docs
https://flet.dev/docs/guides/python/navigation-and-routing

No file found at this location
https://github.com/flet-dev/flet/blob/main/sdk/python/flet/template_route.py

**Comment by ndonkoHenri** [comment]
https://github.com/flet-dev/flet/blob/main/sdk/python/packages/flet-core/src/flet_core/template_route.py
You are right. Here is the file.
